src/cplscheme/BaseCouplingScheme.cpp

```diff
@@ -119,6 +120,8 @@ void BaseCouplingScheme::sendData(const m2n::PtrM2N &m2n, const DataMap &sendDat
   PRECICE_ASSERT(m2n.get() != nullptr);
   PRECICE_ASSERT(m2n->isConnected());
 
+  profiling::Event e("sendData", profiling::Fundamental);
```

Comment:
This is the overall time spend sending.
There can be multiple nested `m2n.send` events.

Reply:
What happens if we have multiple coupling schemes? Should we add the receiver in the name?